cdc/processor/processor.go

```diff
@@ -977,13 +977,6 @@ func (p *processor) createTablePipelineImpl(
 		}
 	}
 
-	log.Info("Add table pipeline", zap.Int64("tableID", tableID),
```

Comment:
Is there any logs about adding table action already?

Reply:
yes, it's known by other log